bundles/action/org.openhab.action.ciscospark/OSGI-INF/action.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" configuration-policy="require" deactivate="deactivate" immediate="true" name="org.openhab.action.ciscospark.action">
```

Comment:
I'm not sure what this means... Admittedly, I'm no osgi/karaf expert.  I've been digging into the docs [here](https://karaf.apache.org/manual/latest/#_configuration_3), but am not seeing any reference to an actual configuration.pid property. The text mentions _the configuration pid_, but appears to refer to `service.pid` as the actual property.  
I'm testing this by manually adding this plugin to the addon directory and the plugin does get loaded and is functional.
If there are changes needed, can give me few pointers/examples of how to proceed?  Thanks for the feedback so far!